spec/src/main/asciidoc/chapters/data-binding.asciidoc

```diff
@@ -0,0 +1,165 @@
+[[data_binding]]
+== Data Binding
+
+This chapter discusses data binding in the MVC API. 
+Data binding is based on the underlying mechanism provided by JAX-RS, 
+but with additional support for I18N requirements and for handling data binding errors within the controller.
```

Comment:
Not to relevant but I think that `i18n` (lowercase) is more common.

Reply:
Thanks, I just updated that.